Changed all label formats to match the original.

DavidJSolano opened the pull request:
This PR fixes issue #655. The new labels look like this: 
![Screenshot_20221120_084817](https://user-images.githubusercontent.com/111948487/202968196-89fd8f65-25dc-47bd-92d6-bb42738de7c6.png)
![Screenshot_20221120_084831](https://user-images.githubusercontent.com/111948487/202968229-ca0831c1-dc5c-4193-bc00-2e562fc00e7e.png)

DavidJSolano committed:
Changed all label formats to match the original.

This PR fixes issue#655.

DavidJSolano committed:
Fixed Security Annotation

Updated the label to contain the rel "noopener noreferrer".

EthanThatOneKid reviewed (approved):
Nice!! Hope you had an epic Thanksgiving break 😎

jaasonw commented on src/lib/components/blog/labels.svelte:
```diff
@@ -34,15 +35,18 @@
-      background-color: var(--acm-light);
-      border-radius: 5px;
-      padding: 0.3em;
+      background-color: #eeeeee;
```
suggest going back to acm-light or using acm-canvas here, avoid having "magic numbers" or mix of different off white throughout the codebase whenever possible

jaasonw replied:
also makes it dark mode reactive

DavidJSolano replied:
Got it.

jaasonw commented on src/lib/components/blog/labels.svelte:
```diff
@@ -34,15 +35,18 @@
-      border-radius: 5px;
-      padding: 0.3em;
+      background-color: #eeeeee;
+      color: #212121;
```
remove this line, for same reasons as above

DavidJSolano replied:
Done.

jaasonw commented on src/lib/components/blog/labels.svelte:
```diff
@@ -34,15 +35,18 @@
+      background-color: #eeeeee;
+      color: #212121;
+      border-radius: var(--size-sm);
+      border: 2px solid #e0e0e0;
```
same reasons as above, maybe experiment if mixing acm-light and acm-canvas would produce a similar effect?
/

DavidJSolano replied:
You got it.

jaasonw commented on src/lib/components/blog/labels.svelte:
```diff
@@ -34,15 +35,18 @@
-      color: var(--perma-light);
+      background-color: #81d4fa;
+      border-color: #4fc3f7;
+      color: #212121;
```
var(--perma-dark) here instead

jaasonw commented on src/lib/components/blog/labels.svelte:
```diff
@@ -34,15 +35,18 @@
     .selected {
-      background-color: var(--acm-gray);
-      color: var(--perma-light);
+      background-color: #81d4fa;
```
how does using var(--acm-blue) and var(--acm-bluer) instead here look?

DavidJSolano replied:
All the other changes look pretty good, not sure if I'm rocking with this change though.
![Screenshot_20221206_011004](https://user-images.githubusercontent.com/111948487/206023739-958cd361-b446-4d37-ade4-b7cd5c02c5fa.png)

DavidJSolano committed:
Changed label colors to match with ACM.

All label colors were replaced with var(--acm-...)

EthanThatOneKid reviewed (approved):
> ![Screenshot_20221206_123739](https://user-images.githubusercontent.com/111948487/206018382-d3e8ba54-3cb1-45d2-9a02-83fb217283d2.png) ![Screenshot_20221206_123754](https://user-images.githubusercontent.com/111948487/206018384-7caddd8f-14ec-4b9b-9e79-dcc7d0436031.png) This is how the new labels look.

Nice! Thanks for also making sure to support dark mode :)

jaasonw reviewed (changes requested):
sorry, i shoulda said this earlier but can you reflect the same changes in labelfield.svelte, or maybe split the label into its own component to avoid duplication?

<img width="472" alt="image" src="https://user-images.githubusercontent.com/8981287/206117183-3ca41c6b-5ebd-49d5-954f-3884550b623e.png">

DavidJSolano committed:
Duplicated Changes

Changed the color of the labels in labelfield.svelte to match acm colors.

DavidJSolano committed:
Reverted selected labels to original colors.

I removed the ACM colors from the .selected labels and reverted them to the original blue look.

DavidJSolano committed:
Merge branch 'main' into orgin/fix/655

DavidJSolano committed:
Light/Dark compatibility with labels.

Changed labelfield.svelte color to be var(--acm--dark) to be compatible with dark/light mode.

DavidJSolano committed:
Merge branch 'orgin/fix/655' of https://github.com/DavidJSolano/acmcsuf.com into orgin/fix/655

jaasonw commented on src/routes/(site)/blog/labelfield.svelte:
```diff
@@ -107,6 +107,7 @@
           transition: 1s ease-in-out;
           background-color: #81d4fa;
           border-color: #4fc3f7;
+          color: var(--acm-dark)
```
looks like you're missing a semicolon here

DavidJSolano replied:
Oops, thanks for pointing that out.

DavidJSolano committed:
Added a semicolon.

Tiny syntactical error.

jaasonw reviewed (approved):
sexy
